<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatroom</class>
 <widget class="QDialog" name="chatroom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>无聊</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 255);</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_chat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>521</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 18pt &quot;Adobe Arabic&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>540</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Adobe Arabic&quot;;
background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>在线账号列表：</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_member">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>80</y>
     <width>171</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 16pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>所有使用者的聊天室</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
